gslib/tests/test_rm.py

```diff
@@ -121,7 +121,7 @@ def _RunRmCommandAndCheck():
         update_lines = True
         # Retry 404's and 409's due to eventual listing consistency, but don't
         # add the output to the set.
-        if (NO_URLS_MATCHED_GENERIC in stderr or
+        if (NO_URLS_MATCHED_PREFIX in stderr or
```

Comment:
ditto

Reply:
Made it more specific by using `NO_URLS_MATCHED_TARGET`

Reply:
Got it. In that case , this should be good.